File: java/soma_linhas.java
import java.util.Locale;
import java.util.Scanner;

public class soma_linhas {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int M, N;
		double somaLinha;

		System.out.print("Qual a quantidade de linhas da matriz? ");
		M = sc.nextInt();
		System.out.print("Qual a quantidade de colunas da matriz? ");
		N = sc.nextInt();

		double[][] mat = new double[M][N];
		double[] vet = new double[M];

		for (int i = 0; i < M; i++) {
			System.out.printf("Digite os elementos da %da. linha\n", i + 1);

			for (int j = 0; j < N; j++) {
				mat[i][j] = sc.nextDouble();
			}
		}

		for (int i = 0; i < M; i++) {			
			somaLinha = 0;
			for (int j = 0; j < N; j++) {
				somaLinha = somaLinha + mat[i][j];
			}
			vet[i] = somaLinha;
		}

		System.out.println("VETOR GERADO:");

		for (int i = 0; i < M; i++) {
			System.out.printf("%.1f\n", vet[i]);
		}

		sc.close();
	}
}

File: python/soma_linhas.py
M: int
N: int
soma: float

M = int(input("Qual a quantidade de linhas da matriz? "))
N = int(input("Qual a quantidade de colunas da matriz? "))

mat: [[float]] = [[0 for x in range(N)] for x in range(M)]
vet: [float] = [0 for x in range(M)]

for i in range(M):
    print(f"Digite os elementos da {i + 1}a. linha")
    for j in range(N):
        mat[i][j] = float(input())

for i in range(M):
    soma = 0
    for j in range(N):
        soma += mat[i][j]
    vet[i] = soma

print("VETOR GERADO:")

for i in range(M):
    print(f"{vet[i]:.1f}")
